checklists/azure_arc_checklist.en.json

```diff
@@ -246,10 +246,11 @@
             "subcategory": "Security",
             "waf": "Operations",
             "text": "Use Azure Arc-enabled servers to control software updates deployments to servers",
-            "description": "Use Update Management in Azure Automation or the new Update Management Center (preview) functionality to ensure update management of servers",
+            "description": "Use Azure Update Manager to manage and schedule updates at scale across your Azure Arc-enabled servers",
             "guid": "ae2cc84c-37b6-4b78-8cba-fe6c46589d45",
             "severity": "Low",
-            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/manage/hybrid/server/best-practices/arc-update-management"
+            "graph": "resources | where type =~ 'microsoft.hybridcompute/machines' | project id = tolower(id), arcMachineName = name | join kind=leftouter (maintenanceresources | extend baseIdParts = split(tolower(id), '/providers/microsoft.maintenance/') | extend maintenanceMachineId = tostring(baseIdParts[0]) | project maintenanceMachineId, maintenanceId = id) on $left.id == $right.maintenanceMachineId | extend compliant = iif(maintenanceMachineId == '', 'No', 'Yes') | project id, compliant",
+            "link": "https://learn.microsoft.com/en-us/azure/update-manager/scheduled-patching?tabs=schedule-updates-single-machine%2Cschedule-updates-scale-overview%2Cwindows-maintenance"
```

Comment:
Hey could you please remove the localization (`en-us/`) from the link? Thanks!

Reply:
Resolved!